src/views/Home.jsx

```diff
@@ -1,59 +1,47 @@
 import React from "react";
-import {
-  HintList,
-  HintListItem,
-  Icon,
-  SubContainer,
-  InstructionText,
-  FormHeader,
-  HintListItemText,
-} from "../components/List";
+import { SubContainer, InstructionText, FormHeader } from "../components/List";
 import { FlexContainer, RightPanel, Card } from "../components/Layout";
 import HeaderText from "../components/text/HeaderText";
-import checkmark from "../components/images/checkmark.svg";
 import theme from "../theme";
 import Anchor from "../components/Anchor";
+import BodyText from "../components/text/BodyText";
 
 const Home = () => {
   return (
     <FlexContainer>
       <SubContainer>
         <HeaderText>Welcome!</HeaderText>
         <InstructionText>
-          We host regular webinars to help answer questions about:
+          Thank you for checkout out our webinar demo.
         </InstructionText>
-        <HintList>
-          <HintListItem>
-            <Icon src={checkmark} />
-            <HintListItemText>
-              Different ways to integrate video quickly with Daily APIs
-            </HintListItemText>
-          </HintListItem>
-          <HintListItem>
-            <Icon src={checkmark} />
-            <HintListItemText>
-              Using your dashboard or /logs endpoint to get call logs and call
-              quality data
-            </HintListItemText>
-          </HintListItem>
-          <HintListItem>
-            <Icon src={checkmark} />
-            <HintListItemText>
-              Anything you’d like to ask about!
-            </HintListItemText>
-          </HintListItem>
-        </HintList>
+        <BodyText>
+          This demo uses both an "admin" view and an "attendee" view. In order
```

Comment:
Change _attendee_ to _participant_ to match the value in the code?

Reply:
I did the opposite and changed all instances of _participant_ to _attendee_. Because in Daily terms, admins/owners are also participants.